server/core/storage.go

```diff
@@ -45,6 +45,8 @@ const (
 	componentPath            = "component"
 	customScheduleConfigPath = "scheduler_config"
 	encryptionKeysPath       = "encryption_keys"
+	// GCWorkerServiceSafePointID is the ID of GCWorker's service GC safe point.
+	GCWorkerServiceSafePointID = "gc_worker"
```

Comment:
How about changing this name to make it private?

Reply:
It's used in test in tests/client/client_test.go .

Reply:
prefer to use "gc_worker" directly in tests